lib/helpers/remote_configuration.rb

```diff
@@ -27,21 +29,40 @@ def f_manager
     @f_manager ||= FileManager.new
   end
 
-  # @param [Net::SSH::Connection::Session] session SSH session
-  #
-  def overwrite_configs(session)
-    dockerfile = ssh.download!(session, StaticData::DOCKERFILE)
-    ssh.upload!(session,
-                StaticData::DOCKERFILE,
-                f_manager.writes_tokens_by_path_array(dockerfile, /""/, StaticData::PATH_ARRAY))
-
-    env = ssh.download!(session, StaticData::ENV)
-    ssh.upload!(session,
-                StaticData::ENV,
-                env = f_manager.overwrite(env, /latest/, f_manager.wrap_in_double_quotes(@version)))
-    ssh.upload!(session,
-                StaticData::ENV,
-                f_manager.overwrite(env, /''/, f_manager.wrap_in_double_quotes(@spec_name)))
+  # @param [Object] tmpdir
+  # @return [Integer]
+  #
+  def overwrite_dockerfile(tmpdir)
```

Comment:
I think sending full path to `Dockerfile` in this method will be more precise, than sending dir and *hoping* that there is a Dockerfile inside

Same for `dot_env`